ada_2_fhir-r4/cio/2.0.0/payload/cio-ReplyProposalContraIndication.xsl

```diff
@@ -0,0 +1,138 @@
+                <xsl:variable name="creationDateTime"
+                    select="$registrationInformation/ontstaans_datum_tijd"/>
+
+                <xsl:call-template name="insertLogicalId">
+                    <xsl:with-param name="profile"
+                        select="$profileNameCioReplyProposalContraIndication"/>
+                </xsl:call-template>
+                <meta>
+                    <profile
+                        value="{concat($urlBaseNictizProfile, $profileNameCioReplyProposalContraIndication)}"
+                    />
+                </meta>
+
+                <xsl:for-each select="$identificationNumber[@value]">
+                    <identifier>
+                        <xsl:call-template name="id-to-Identifier">
+                            <xsl:with-param name="in" select="."/>
+                        </xsl:call-template>
+                    </identifier>
+                </xsl:for-each>
+
+                <xsl:for-each select="relatie_voorstel_contra_indicatie/identificatie[@value | @root | @nullFlavor]">
+                    <basedOn>
+                        <valueReference>
+                            <type value="Flag"/>
+                            <identifier>
+                                <xsl:call-template name="id-to-Identifier">
+                                    <xsl:with-param name="in" select="."/>
+                                </xsl:call-template>
+                            </identifier>
+                            <display value="Relatie naar voorstel: {string-join((@value, @root), ' || ')}"/>
+                        </valueReference>
+                    </basedOn>
+                </xsl:for-each>
+
+                <xsl:for-each select="$creationDateTime[@value]">
+                    <sent>
+                        <xsl:attribute name="value">
+                            <xsl:call-template name="format2FHIRDate">
+                                <xsl:with-param name="dateTime" select="xs:string(@value)"/>
+                            </xsl:call-template>
+                        </xsl:attribute>
+                    </sent>
+                </xsl:for-each>
+
+                <xsl:for-each select="$author">
+                    <xsl:call-template name="makeReference">
+                        <xsl:with-param name="in" select="."/>
+                        <xsl:with-param name="wrapIn" select="'author'"/>
+                        <xsl:with-param name="profile"
+                            select="$profileNameHealthProfessionalPractitionerRole"/>
+                    </xsl:call-template>
+                </xsl:for-each>
+                
+                <xsl:for-each select="beantwoording_voorstel_contra_indicatie[@value]">
+                    <payload>
+                        <contentString>
+                            <extension url="http://hl7.org/fhir/5.0/StructureDefinition/extension-Communication.payload.content">
```

Comment:
This url should be changed to the custom extension defined by Nictiz, since we unfortunately cannot use the core package containing R5 pre-adopts as extensions.